src/windows/editor_scenario_options.c

```diff
@@ -559,7 +559,7 @@ static void window_editor_scenario_options_financial_mousedown(int widgetIndex,
 		break;
 	}
 
-	if(RCT2_GLOBAL(RCT2_ADDRESS_SCREEN_FLAGS, uint8) & SCREEN_FLAGS_PLAYING) {
+	if(RCT2_GLOBAL(RCT2_ADDRESS_SCREEN_FLAGS, uint8) == SCREEN_FLAGS_PLAYING) {
 		window_invalidate_by_class(WC_FINANCES);
```

Comment:
This was a mistake made in 0491d971f3ad79e6327b3fc14d8180241ca5a107. This is right as it should only invalidate the finance related windows if you are changing things in this window in-game.

See:
https://github.com/janisozaur/OpenRCT2/commit/0491d971f3ad79e6327b3fc14d8180241ca5a107#diff-31802840e0eb7d8cc0b1347e0883b20bR569